### DISCUSSION TITLE
CVAR to enable/disable options in "Team Vote Menu"

### DISCUSSION THREAD

**Discussion by Haco** [SmileYzn/PugMod-AMXX#34] [Ideas]
Could you introduce a new cvar to control ( enable or disable ) the options in the "Team Vote Menu"? 

Currently there are those options:

1. Leaders Sorted
2. Random
3. Not Sorted
4. Skill Sorted
5. Swap Teams
6. Knife Round

Then the server admin could customize the available options by a cvar setting. Like this, for example:

1. Leaders Sorted
2. Random
3. Skill Sorted

Thx

**Comment by gameinside** [comment]
Xm... Nice ideas

**Comment by SmileYzn** [comment]
This can cause some headcache to me :rofl:   
Duo that menus in vote are sequentially, and indexes are required to create menus.  

Also the "cvar" need to be created in a way that is possible to determine what options will be on menu like:  

pug_teams_enforcement_menu "1,2,3,4,5,6" (Or "123456") ??
